src/bench/rollingbloom.cpp

```diff
@@ -13,16 +13,16 @@ static void RollingBloom(benchmark::Bench& bench)
     uint32_t count = 0;
     bench.run([&] {
         count++;
-        data[0] = count;
-        data[1] = count >> 8;
-        data[2] = count >> 16;
-        data[3] = count >> 24;
+        data[0] = static_cast<unsigned char>(count);
+        data[1] = static_cast<unsigned char>(count >> 8);
+        data[2] = static_cast<unsigned char>(count >> 16);
+        data[3] = static_cast<unsigned char>(count >> 24);
         filter.insert(data);
 
-        data[0] = count >> 24;
-        data[1] = count >> 16;
-        data[2] = count >> 8;
-        data[3] = count;
+        data[0] = static_cast<unsigned char>(count >> 24);
+        data[1] = static_cast<unsigned char>(count >> 16);
+        data[2] = static_cast<unsigned char>(count >> 8);
+        data[3] = static_cast<unsigned char>(count);
```

Comment:
I can't easily change that because the RollingBloomHash's API uses that and I don't want the change to be too large. I've changed this though to use `& 0xFF` like it's done in many other places in the code. Done in b901174bfb6895c8e332c2b5e280348836398e68